src/fparser/two/Fortran2003.py

```diff
@@ -1263,25 +1255,88 @@ def restore_reader(self, reader):
         reader.put_item(self.item)
 
 
+def add_comments(content, reader):
+    '''Creates comment objects and adds them to the content list. Comment
+    objects are added until a line that is not a comment is found.
+
+    :param content: a `list` of matched objects. Any matched comments
+                    in this routine are added to this list.
+    :param reader: the fortran file reader containing the line(s)
+                   of code that we are trying to match
+    :type reader: :py:class:`fparser.common.readfortran.FortranFileReader`
```

Comment:
Please use "\" to indicate line breaks in the doc strings.

Reply:
I've added "\" to the end of the lines that are continued. Is that OK?